### PR TITLE
Custom Resource Validation: remove alpha warning, change enable instructions to disable

### PR DESCRIPTION
**Pull request by steveperry-53**
@colemickens 
Duplicates #6065.
Per this issue, we are promoting Custom Resource Validation to beta in 1.9: kubernetes/kubernetes#53829

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6066)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@colemickens If this looks OK to you, I'll merge it into the release-1.9 branch.

**Comment by colemickens** [comment]
@steveperry-53, can I just correct the PR I already have open for this? Not sure why you opened a new one?

**Comment by steveperry-53** [comment]
@colemickens Yes, if you prefer, you can fix #6065 and close #6066.

**Comment by colemickens** [comment]
I guess I see what happened. It's fine either way - I rebased my branch if you want to reopen and merge my PR. If you want to merge yours that's fine! Thanks!

One thing though, maybe we should hold off merging until the actual promotion goes into main tree.

**Comment by steveperry-53** [comment]
@colemickens We can merge this into release-1.9. The content in release-1.9 won't be public until the Kubernetes 1.9 product is released at the end of December (or early January). Does that sound OK?

File: docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md [hunk 1]
```diff
@@ -190,12 +190,12 @@ Additionally, the following restrictions are applied to the schema:
 - The field `uniqueItems` cannot be set to true.
 - The field `additionalProperties` cannot be set to false.
 
-This feature is __alpha__ in v1.8 and may change in backward incompatible ways.
-Enable this feature using the `CustomResourceValidation` feature gate on
+This feature is __beta__ in v1.9.
```
Comment (nikhita) [inline docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md]: `from v1.9` instead of `in v1.9`.

**Comment by nikhita** [comment]
> The content in release-1.9 won't be public until the Kubernetes 1.9 product is released at the end of December (or early January). Does that sound OK?

@steveperry-53 technically, this makes perfect sense! But it feels "righter" to have the docs merged after the feature has _really_ moved to beta. :smile: 

The corresponding PR https://github.com/kubernetes/kubernetes/pull/54647 should be merged soon though!

**Comment by nikhita** [comment]
Copying my comment from https://github.com/kubernetes/kubernetes/pull/54647#issuecomment-339825611,

> IMO this should be called "CustomResource validation" instead of "CRD Validation" (especially in the release note). We define the schema in the CRD but we are validating the custom resources, not the CRD itself.

This doesn't affect the docs but would be nice to have the PR title reflect the right nomenclature (so that it's easier to search in the future). :)

**Comment by nikhita** [comment]
Also, maybe we should mention the `beta` statement in the beginning and then explain what the feature is really about. WDYT?

File: docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md (hunk 1 above)
Comment (colemickens) [reply docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md]: Split the difference? "This feature is beta beginning with v1.9" ?

**Comment by colemickens** [comment]
Sure, let me add some more descriptive text! And agree'd on the PR title, will fix in all places.

**Comment by colemickens** [comment]
Except that I can't change the title now because this is no longer my PR. :/

File: docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md (hunk 1 above)
Comment (nikhita) [reply docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md]: Yup, this sounds better! :+1:

**Comment by nikhita** [comment]
@steveperry-53 Can you make the changes? Thanks!

**Comment by colemickens** [comment]
Hi @steveperry-53. I'm fine with your PR being merged. Thanks! I'm happy to pull and rebase in the future :).

Let's hold off until the CI goes green and this actually gets merged in the main repo though: https://github.com/kubernetes/kubernetes/pull/54647

**Comment by colemickens** [comment]
@steveperry-53 This is good to merge, please. Thanks.

**Comment by colemickens** [comment]
On second thought, I've got my wires crossed and this PR isn't in yet. Hold off a bit more.

**Comment by colemickens** [comment]
It merged: https://github.com/kubernetes/kubernetes/pull/54647#event-1345314054.

Can you merge this @steveperry-53 ? Thanks!

**Comment by zacharysarah** [comment]
LGTM 👍

**Pull request merged**